qemu/tests/cfg/block_iscsi_lvm.cfg

```diff
@@ -0,0 +1,87 @@
+        lv_size = 20G
+    Windows:
+        lv_size = 30G
+    block_raw_name = sys
+    emulated_image = images/${block_raw_name}
+    Linux:
+        emulated_image_size = 21G
+    Windows:
+        emulated_image_size = 31G
+    images = 'image0'
+    image_verify_bootable = no
+    force_create_image0 = no
+    image_format_image0 = raw
+    image_name_image0 = /dev/${vg_name}/${lv_name}
+    image_raw_device_image0 = yes
+    remove_image_image0 = no
+    cmd_fdisk = "echo -e "g\nn\n1\n\n\nw\n" | fdisk %s"
+    cmd_get_partition = "fdisk -l {0} | grep {0}[1-9] | awk '{{print $1}}'"
+    cmd_remove_lv = "lvm lvremove -ff --yes ${image_name_image0}"
+    cmd_remove_vg = "lvm vgremove -ff --yes ${vg_name}"
+    cmd_remove_pv = "lvm pvremove -ff --yes %s"
+    cmd_delete_iscsi_block = "name=`targetcli /backstores/fileio/ ls | grep ${block_raw_name} | awk '{print $2}'` && "
+    cmd_delete_iscsi_block += "targetcli /backstores/fileio/ delete $name"
+    cmd_get_iscsi_iqn = "iqn=`targetcli /iscsi/ ls | grep ${target} | awk '{print $2}'`"
+    cmd_delete_luns = "targetcli /iscsi/$iqn/tpg1/luns/ delete lun=0"
+    cmd_delete_iqn = "targetcli /iscsi/ delete $iqn"
+    cmd_remove_raw_block = "rm -rf %s"
+    cmd_remove_dev_vg = "rm -rf /dev/${vg_name}"
+    cmd_iscsi_logout = "iscsiadm --mode node --logout -T ${target}"
+    cmd_iscsi_delete_node = "iscsiadm -m node -o delete -T ${target} --portal 127.0.0.1"
+    cmd_iscsi_mv_config = "mv /etc/iscsi/initiatorname.iscsi-%s /etc/iscsi/initiatorname.iscsi"
+    cmd_iscsi_restart = "systemctl reset-failed iscsid.service && systemctl restart iscsid.service"
+    post_commands_iscsi = "${cmd_remove_dev_vg},${cmd_remove_raw_block},${cmd_iscsi_restart},"
+    post_commands_iscsi += "${cmd_iscsi_mv_config},${cmd_delete_iscsi_block},"
+    post_commands_iscsi += "${cmd_delete_iqn},${cmd_delete_luns},${cmd_get_iscsi_iqn},"
+    post_commands_iscsi += "${cmd_iscsi_delete_node},${cmd_iscsi_logout}"
+    post_commands_lvm = "${cmd_remove_pv},${cmd_remove_vg},${cmd_remove_lv}"
+    post_command_noncritical = no
+    variants:
+        - with_scsi_hd:
+            drive_format_image0 = scsi-hd
+            Windows:
+                blk_extra_params_image0 = "serial=TARGET_DISK0"
+        - with_scsi_block:
+            drive_format_image0 = scsi-block
+    variants:
+        - with_installation:
+            cd_format_cd1 = scsi-cd
+            i440fx:
+                Windows:
+                    cd_format_unattended = ide
+            q35:
+                cd_format_unattended = ahci
+            variants:
+                - extra_cdrom_ks:
+                    no WinXP Win2000 Win2003 WinVista
+                    cdroms += " unattended"
+                    drive_index_unattended = 3
+                    drive_index_cd1 = 1
+            variants:
+                - cdrom:
+                    Windows:
+                        cdroms += " winutils"
+                        cd_format_winutils = scsi-cd
```

Comment:
Hi yhong, currently, all scsi-cd drive are connected to the same virtio-scsi-pci bus, if virtio-scsi-pci enabled dataplane, then all scsi-cd will cannot connect on due to not support,  in order to match dataplane=on situation, better here change cd_format_winutils/cd1 = ide.

Reply:
ok , will update it.